hydrosdk/utils.py

```diff
@@ -1,18 +1,56 @@
-import grpc
+from typing import Generator
 
-ALLOWED_TIMEOUT_gRPC_SECONDS = 5
```

Comment:
why deleting default timeout?

Reply:
Because @tidylobster have moved timeout to the `Cluster` constructor